core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.ts

```diff
@@ -69,6 +69,7 @@ angular.module('oppia').component('subtopicEditorTab', {
             QuestionBackendApiService.fetchTotalQuestionCountForSkillIds(
               ctrl.skillIds).then((questionCount) => {
               ctrl.questionCount = questionCount;
+              $scope.$applyAsync();
```

Comment:
Why was this not required earlier?

Reply:
It's behvaiour was flaky, Hence added this just to be sure.

Reply:
Replying to this [comment](https://github.com/oppia/oppia/pull/9954#issuecomment-661022835):

Thanks @Showtim3 for the video! I understand that this fixes an issue related to rendering the page. But I wanted you to explain why this is the correct fix for the issue (see this [comment](https://github.com/oppia/oppia/pull/9916#issue-449742664) for a similar fix). A detailed comment like that will help other reviewers understand why this change was done.